pypeit/pypeit.py

```diff
@@ -1142,3 +1144,276 @@ def __repr__(self):
         return '<{:s}: pypeit_file={}>'.format(self.__class__.__name__, self.pypeit_file)
 
 
+class JWST(PypeIt):
```

Comment:
I've removed this for now and deferring the full JWST pipeline to a future PR.  JWST cannot be reduced in a similar fashion as regular PypeIt spectrographs since there are no calibrations and so JWST pipeline outputs must be ingested. However, ,much of the pypeit file parsing code is the same and so a separate module would have involved replicating many lines of code. Anyway, it is worth discussing the best path forward here, but the pypeit.py class needs to be restructured because it is doing way too many things and is too complicated.